assets/src/edit-story/output/page.js

```diff
@@ -29,27 +29,38 @@ function OutputPage({ page }) {
     height: `calc(100 * ${PAGE_HEIGHT / PAGE_WIDTH} * var(--story-page-vw))`, // 16/9 * 100vw
     maxHeight: `calc(100 * var(--story-page-vh))`, // 100vh
     maxWidth: `calc(100 * ${PAGE_WIDTH / PAGE_HEIGHT} * var(--story-page-vh))`, // 9/16 * 100vh
-    // todo@: this expression uses CSS `min()`, which is still very sparsely
-    // supported.
+    // todo@: this expression uses CSS `min()`, which is still very sparsely supported.
     fontSize: `calc(100 * min(var(--story-page-vh), var(--story-page-vw) * ${PAGE_HEIGHT /
       PAGE_WIDTH}))`,
   };
-  const backgroundElements = page.elements.filter(
-    (element) => element.id === page.backgroundElementId
+  const backgroundNonFullbleedElements = page.elements.filter(
+    (element) =>
+      element.id === page.backgroundElementId &&
+      element.isFullbleedBackground === false
```

Comment:
Are we certain it's always set? Maybe just run with `Boolean(isFullbleedBackground)` here and below?

Reply:
It's not always set. I've implemented it as follows:

* If missing (`undefined`/`null`): use default behavior -> fullbleed
* If explicitly set to `true`: use fullbleed
* If explicitly set to `false`: no fullbleed

Hence the explict checks for `false` here and below.

Alternatives would be setting it explicitly in `setBackgroundElement`, and adding a migration to add it to pre-existing background elements.

Open to suggestions.

Reply:
Merging this one now, but happy to follow-up in another PR